client/src/pages/Leaderboard.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import Header from "../components/Header";
+
+class Leaderboard extends React.Component {
+  render() {
+    return (
+      <div>
+        <div className="Title">Wassup dudes</div>
```

Comment:
This should be a lowercase "title" className to be consistent

Reply:
got eem